Assets/Code/Scripts/PlayerManagement/TimeConstraints.cs

```diff
@@ -16,6 +16,8 @@ public class TimeConstraints : MonoBehaviour
         private float _realityTimeLeftInThisPuzzle;
         private bool _isRunning;
 
+        private float _fadeTime = 1.0f;
```

Comment:
```suggestion
        private float _fadeTime;
```

Really minor: I would not initialize it, as it must be equal to the value that we read in `Start`